app/views/loan_documents/edit.slim

```diff
@@ -0,0 +1,37 @@
+.container-fluid
```

Comment:
@ericksonmc There are two ways of handling this in a rails way:

1. This is an index page, that comes from the loan detail, and shows the loan documents list, and then with a button to upload, takes me to a new one, etc. Normal CRUD, spanned thru several pages.

2. If you wanna show the loan information with the documents information in one screen, then we use the javascript way and render the loan documents index as the forms as partials in a father view.

The approach presented here is none of them, and it's mixing on an edit template the information of loans#show action, and the documents edit stuff.

This application is too small to deviate from the Rails standards and generate extra complexity by mixture and duplication. We need to refactor this.